Makefile

```
@@ -7,6 +7,8 @@ PYTEST_ARGS=--cov=$(PROJECT) --cov-config=../.coveragerc \
 			--pyargs ${PYTEST_EXTRA}
 BLACK_FILES=$(PROJECT) setup.py doc/conf.py examples
 BLACKDOC_OPTIONS=--line-length 79
+DOCFORMATTER_FILES=$(PROJECT) setup.py doc/conf.py examples
+DOCFORMATTER_OPTIONS=--recursive --wrap-summaries 79 --wrap-descriptions 79
```

The default lengths are 79 chars for summary and 72 chars for description. Here I choose 79 for both. There are also some options to add more blank lines before/after summary/description. Not sure if we need them.

---

```suggestion
DOCFORMATTER_OPTIONS=--recursive --pre-summary-newline --wrap-summaries 79 --wrap-descriptions 79
```
Add this to reduce the diff size (Maintains the newline after the triple quotes), and also looks better IMO.

> The default lengths are 79 chars for summary and 72 chars for description. Here I choose 79 for both.

Sounds good. Took me a while to understand the difference between a summary and a description.

> There are also some options to add more blank lines before/after summary/description. Not sure if we need them.

Can probably skip this, prefer to keep things fairly short.

---

All are good points!